networkconfig/local-testnet.go

```diff
@@ -0,0 +1,24 @@
+package networkconfig
+
+import (
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv/protocol/v2/blockchain/beacon"
+	"math/big"
+)
+
+var LocalNetwork = NetworkConfig{
+	Name:                 "local-network",
```

Comment:
What do you think about naming it `local-testnet` like the file name?

Reply:
I agree lets keep it consistent